### PR TITLE
Add New Site: celebrate if simple site

### LINKED ISSUE #32400: Add New Site: Dead end error when adding a new Jetpack site
Screen_Shot_2019-03-06_at_11 58 34_AM

As a user unfamiliar with Jetpack (and who didn't read / understand the page sub-header copy) I entered the URL for the site I just created and got this error "That's a WordPress.com site, so you don't need to connect it." I can dismiss the error, but the page remains, which can be very confusing.

It works to just use the browser back button, but we should help the user get back to the previous screen or simplify by removing or minimizing Jetpack options.

This could be a moment to celebrate/highlight -- "You already have Jetpack's great features!" (Something more eloquent than that, but you get the idea) -- rather than a bright red, off-putting error.

### PR DESCRIPTION
**Pull request by donpark**
#### Changes proposed in this Pull Request

Implements messaging part of #32400, replacing error message with a positive feedback.

#### Testing instructions

1. If testing with local Calypso build, navigate to Add New Site flow initial step page. With calypso.localhost, it's http://calypso.localhost:3000/jetpack/new?ref=calypso-selector

2. Enter a Simple site URL like https://foobar.wordpress.com and Continue.

3. Close the notice by clicking on the X button at top right corner of the notice.

Expected:

add-new-site-dead-end

Fixes #32400

### CODE REVIEW

**Comment by donpark** [comment]
Requested copy review.

Wording has changed from:

> That's a WordPress.com site, so you don't need to connect it.

to

> That's a WordPress.com site, always connected!

Short and sweet yet lacking something I think.

**Comment by sixhours** [comment]
Yay! I'll throw my .02 into the ring:

> "That's a WordPress.com site, so you're already connected!"

Probably too wordy. /cc @michelleweber and @obi2020 for thoughts from editorial and content-wrangling perspectives.

**Comment by michelleweber** [comment]
I think Caroline's option works. Some others (but note that I am wordy, so I defer to Obi):

That's a WordPress.com site -- no connection needed!

You're already set -- that's a WordPress.com site. 

You're all set -- that's a WordPress.com site, so Jetpack's benefits are already baked in.

**Comment by obi2020** [comment]
Seems like you two are on the right track @sixhours @michelleweber. My Jetpack knowledge is small, so the only direction I can give here is short is better. I like your idea to use an emdash, Michelle. Here are some more ideas

That's a WordPress.com site -- Jetpack is already connected 👍

Good news! WordPress.com sites already have Jetpack ✅

**Comment by sixhours** [comment]
Thanks for the speedy feedback!

@donpark, I think we should go with "Good news! WordPress.com sites already have Jetpack."

It sounds like the navigation piece would need to be worked on as a separate PR, although I am personally OK with a hacky fix in the meantime. :)

**Comment by donpark** [comment]
> I think we should go with "Good news! WordPress.com sites already have Jetpack."

Done.

> It sounds like the navigation piece would need to be worked on as a separate PR, although I am personally OK with a hacky fix in the meantime. :)

@sixhours 'hacky' fix is in. To test, start one step back which, for calypso.localhost is:
http://calypso.localhost:3000/jetpack/new?ref=calypso-selector

**Comment by donpark** [comment]
I've updated test instruction and replaced screenshot with animated GIF showing the expected visuals.
